__mocks__/svgMock.js

```diff
@@ -0,0 +1 @@
+export default 'SvgMock';
```

Comment:
Out of curiosity, why are we adding this?

Reply:
![image](https://github.com/user-attachments/assets/37302ec5-dca7-4181-a6f2-1a403a5245d3)

without it - the test blows up essentially, with this error

![image](https://github.com/user-attachments/assets/2ad94108-9630-4a6a-a080-a020c0bc54a6)

the __mocks__ folder is the default used by jest, when paired with the change to the jest config it fixes the issue with svgs breaking tests

Reply:
Did we know why it started breaking? Was it due to a version update or similar?

Reply:
it breaks when you attempt to test a file that contains an svg. If you check guide-client where we have many more tests, you can see that the jest config already contains a  moduleNameMapper for svg

Reply:
Was just wondering why we didn't see the error before. But I see that it might be because of the new tests we are adding as well. Thank you for clarifying.